### ISSUE TITLE
dev-release/3.0.2

### ISSUE THREAD

**Issue by Froxz** [Pseudoagentur/soa-sentinel#126]
1. Missing  ```js vendor/sleeping-owl/admin/default/plugins/bootstrap-datetime-picker/js/datetime/s_bootstrap-datetimepicker.min.js Failed to load resource: the server responded with a status of 404 ``` the folder structure is bootstrap-datetime-picker/js/bootstrap-datetimepicker.min.js
2. Not translating datatables (search, rows to show), also in menu.php not possible to use trans_choice

**Comment by Froxz** [comment]
```php
'custom_routes' => [
	'admin.my_route' => 'myroute'
]
```
Not hiding URL from Menu

**Comment by Froxz** [comment]
superadmin permission
if superadmin is true, the menu is shown but cant create edit or delete etries, only view.

**Comment by Froxz** [comment]
User has superadmin:true
The menu links are shown, can view entries, but buttons are not displayed for create, edit, delete

```php
\Admin::model('SleepingOwl\Admin\Model\Permission')->title('Permissions')->display(function ()```

**Comment by Froxz** [comment]
```php
FormItem::iradio('awesomefield', 'Radio Label'),
```
doesn't work

**Comment by noxify** [comment]
```
User has superadmin:true
The menu links are shown, can view entries, but buttons are not displayed for create, edit, delete

    \Admin::model('SleepingOwl\Admin\Model\Permission')->title('Permissions')->display(function ()

    I can access create form by typing url /dashboar/permissions/create
```

should be fixed

**Comment by noxify** [comment]
there was an issue in the permission calculation

**Comment by noxify** [comment]
iradio works fine :) You forgot to add the radiobutton definition ;)

```
FormItem::iradio('awesomefield', 'Radio Label')->options([1 => 'First', 2 => 'Second', 3 => 'Third'])
```

**Comment by noxify** [comment]
```
Missing js vendor/sleeping-owl/admin/default/plugins/bootstrap-datetime-picker/js/datetime/s_bootstrap-datetimepicker.min.js Failed to load resource: the server responded with a status of 404 the folder structure is bootstrap-datetime-picker/js/bootstrap-datetimepicker.min.js
```

fixed

**Comment by noxify** [comment]
```
Not translating datatables (search, rows to show), also in menu.php not possible to use trans_choice
```

fixed

merged develop with 3.0.2 to have all changes also in the dev env.

If you found more bugs, please create a new issue/reopen this issue.
